Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/day/tabs/common/DiaryCircumstancesTextView.kt

```diff
@@ -0,0 +1,66 @@
+package de.rki.coronawarnapp.contactdiary.ui.day.tabs.common
+
+import android.content.Context
+import android.text.Editable
+import android.text.TextWatcher
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.EditText
+import android.widget.ImageView
+import androidx.constraintlayout.widget.ConstraintLayout
+import de.rki.coronawarnapp.R
+
+class DiaryCircumstancesTextView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : ConstraintLayout(context, attrs, defStyleAttr) {
+
+    private val input: EditText
+    private val infoButton: ImageView
+
+    private var afterTextChangedListener: ((String) -> Unit)? = null
+    private val circumStanceTextWatcher = object : TextWatcher {
```

Comment:
Instead of `TextWatcher` with empty method implementations, you could use `TextView.doAfterTextChanged` from KTX: https://developer.android.com/kotlin/ktx/extensions-list#extension_functions_18

Reply:
Using the focus approach for now, but thanks will use next time!